bimmer_connected/vehicle/vehicle.py

```diff
@@ -4,7 +4,14 @@
 from typing import TYPE_CHECKING, Dict, List, Optional, Tuple, Type
 
 from bimmer_connected.api.client import MyBMWClient
-from bimmer_connected.const import ATTR_ATTRIBUTES, ATTR_CAPABILITIES, ATTR_STATE, VEHICLE_IMAGE_URL, CarBrands
+from bimmer_connected.const import (
+    ATTR_ATTRIBUTES,
+    ATTR_CAPABILITIES,
+    ATTR_CHARGING_SETTINGS,
+    ATTR_STATE,
+    VEHICLE_IMAGE_URL,
+    CarBrands,
+)
 from bimmer_connected.models import StrEnum, ValueWithUnit
```

Comment:
Import the function to check json strings:

`from bimmer_connected.utils import is_json`